### DISCUSSION TITLE
Option to Run HiRes. Fix after Quality Gate

### DISCUSSION THREAD

**Discussion by pto2k** [AIrjen/OneButtonPrompt#88] [Ideas]
Hi,
In OBRAU, Hires.fix takes much more time than the initial generation.(it is like 5s/it vs 4it/s )
So I think it could save lots time to run a batch of generations first, then do Quality Gate selection, then run Hires. fix on the winner with rest workflow.  Does that make sense?
Thanks.

**Comment by AIrjen** [comment]
Hi yes! Saw your idea on reddit as well. Looks doable, will put it on the todo list.

**Comment by pto2k** [comment]
Thanks!

**Comment by AIrjen** [comment]
New options are in! Based on your suggestions.

![image](https://github.com/AIrjen/OneButtonPrompt/assets/130234949/e519795a-dbd5-4dc7-b96c-8f4975498ec2)

Have fun ;)

**Comment by pto2k** [comment]
This works so great! 

![00270-20230722151317_art by Cornelis Springer, Still Life, Great snow robot, Cubism, art by Pete Turner, Aestheticism_None_2122529570](https://github.com/AIrjen/OneButtonPrompt/assets/4364117/ab34aae9-da07-40af-b0a9-24434db48991)
